ci/jenkins/Jenkins_steps.groovy

```diff
@@ -515,6 +515,51 @@ def compile_windows_cpu() {
     }]
 }
 
+def compile_windows_cpu_mkldnn() {
+    return ['Build CPU MKLDNN windows':{
+      node(NODE_WINDOWS_CPU) {
+        ws('workspace/build-cpu-mkldnn') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.init_git_win()
+            powershell 'py -3 ci/build_windows.py -f WIN_CPU_MKLDNN'
+            stash includes: 'windows_package.7z', name: 'windows_package_cpu_mkldnn'
+          }
+        }
+      }
+    }]
+}
+
+def compile_windows_cpu_mkldnn_mkl() {
+    return ['Build CPU MKLDNN MKL windows':{
+      node(NODE_WINDOWS_CPU) {
+        ws('workspace/build-cpu-mkldnn-mkl') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.init_git_win()
+            //powershell '($Env:MKL_ROOT="C:\Program Files (x86)\IntelSWTools\compilers_and_libraries\windows\mkl") -and (py -3 ci/build_windows.py -f WIN_CPU_MKLDNN_MKL)'
+            powershell 'py -3 ci/build_windows.py -f WIN_CPU_MKLDNN_MKL'
+			stash includes: 'windows_package.7z', name: 'windows_package_cpu_mkldnn_mkl'
+          }
+        }
+      }
+    }]
+}
+
+def compile_windows_cpu_mkl() {
+    return ['Build CPU MKL windows':{
+      node(NODE_WINDOWS_CPU) {
+        ws('workspace/build-cpu-mkl') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.init_git_win()
+            powershell '$Env:MKL_ROOT="C:\Program Files (x86)\IntelSWTools\compilers_and_libraries\windows\mkl"'
```

Comment:
Could you move this code into build_windows.py? We prefer to have these scripts self-contained and the Jenkinsfile just being an orchestration tool.